hugegraph-api/src/main/java/com/baidu/hugegraph/api/auth/ProjectAPI.java

```diff
@@ -68,7 +70,8 @@ public String create(@Context GraphManager manager,
 
         HugeGraph g = graph(manager, graph);
         HugeProject project = jsonTarget.build();
-        project.id(manager.authManager().createProject(project));
+        Id projectId = manager.authManager().createProject(project);
+        project = manager.authManager().getProject(projectId);
```

Comment:
comment why need to get again

Reply:
opGroupId, adminGroupId. etc, these field been knowned only after project created

Reply:
add comment to code